### ISSUE TITLE
Jest docblocks not being preserved in TSX files on Wallaby startup

### ISSUE THREAD

**Issue by jgoz** [wallabyjs/public#1335]
### Issue description or question

Related to #1216. `jest-environment` docblocks are not being preserved in `.tsx` files until the file is modified _after_ Wallaby starts.

Original `.tsx` file:

```ts
/**
 * @jest-environment jsdom
 */

import * as enzyme from "enzyme";
```

Instrumented `.js` file after Wallaby first starts:

```js
'use strict';
$_$wp(109);
$_$w(109, 0), Object.defineProperty(exports, '__esModule', { value: true });
var enzyme = ($_$w(109, 1), require('enzyme'));
```

Instrumented `.js` file after it is touched/modified while Wallaby is running:

```js
/**  * @jest-environment jsdom  */'use strict';
$_$wp(109);
$_$w(109, 0), Object.defineProperty(exports, '__esModule', { value: true });
```

This is reproducible consistently in my environment. I could create a repro solution if that would help.

### Wallaby.js configuration file

```javascript
odule.exports = {
    files: [
        "jest.config.js",
        "tsconfig.json",
        // source files
    ],
    tests: [
        // test files
    ],
    env: {
        type: "node",
        runner: "node",
        params: {
            env: "NODE_ENV=test"
        }
    },
    testFramework: "jest",
    setup: function (wallaby) {
        const jestConfig = require("./jest.config");
        wallaby.testFramework.configure({
            setupTestFrameworkScriptFile: jestConfig.setupTestFrameworkScriptFile,
            testEnvironment: jestConfig.testEnvironment,
            testMatch: jestConfig.testMatch
        });
    }
};
```

### Jest config file

```js
module.exports = {
    moduleFileExtensions: ["js", "json", "ts", "tsx"],
    setupTestFrameworkScriptFile: "<rootDir>/lib/jest/setup.js",
    testEnvironment: "node", // **node by default, not jsdom**
    testMatch: [
        // patterns
    ],
    testResultsProcessor: "jest-teamcity",
    transform: {
        ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
    }
};
```

### TypeScript config

```json
{
  "compileOnSave": false,
  "compilerOptions": {
    "importHelpers": true,
    "jsx": "react",
    "lib": [
      "dom",
      "es5",
      "es2015.collection",
      "es2015.promise"
    ],
    "module": "commonjs",
    "moduleResolution": "node",
    "noImplicitReturns": true,
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "preserveConstEnums": true,
    "preserveSymlinks": true,
    "removeComments": false,
    "sourceMap": true,
    "strict": true,
    "target": "es5"
  }
}
```

### Code editor or IDE name and version

Visual Studio Code v1.16.1

### OS name and version

OSX 10.13

**Comment by ArtemGovorov** [comment]
@jgoz Thanks for the report. Apparently #1216 was not fully fixed. The issue is fixed now and the fix is published in [the latest core](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update) version.

**Comment by ArtemGovorov** [comment]
Forgot to mention, after the core update, please edit your wallaby config (even a space anywhere in it would do) to reset wallaby project cache.

**Comment by jgoz** [comment]
@ArtemGovorov Updated locally to core v1.0.502, forcibly deleted project cache, and I'm still experiencing the issue, exact symptoms as reported in the description.

Is there anything else I can provide to help diagnose this?

**Comment by ArtemGovorov** [comment]
@jgoz Oops, sorry, apparently the commit with the fix didn't make it to v1.0.502. Just released [v1.0.503](https://wallabyjs.com/docs/intro/troubleshooting.html#forcing-wallabyjs-core-update) with the fix.

**Comment by jgoz** [comment]
Perfect, all ✅. Thank you for the speedy turnaround!
